### ISSUE TITLE
Upgrade to OpenBSD 6.9 and use full sets?

### ISSUE THREAD

**Issue by ZhanYF** [cross-platform-actions/openbsd-builder#1]
Hi, since 6.9 has been released for a few months and 7.0 is now in beta, I think it would be a good idea to upgrade this builder to the latest version. I did that on my own branch and the result images look good. However I'm unable to test it properly with GitHub action as I'm unable to compile the required Typescript bundle. Could you help me test that?

Apart from that, I believe X11 should not be left out as it is required for building packages, and it can cause problems if the CI environment is different from the official builder's environment.

https://www.openbsd.org/faq/ports/ports.html#PortsIntro
> Building ports on systems without X11 is not supported. 

Any idea? :)

**Comment by jacob-carlborg** [comment]
> Hi, since 6.9 has been released for a few months and 7.0 is now in beta, I think it would be a good idea to upgrade this builder to the latest version.

That shouldn't be a problem.

> However I'm unable to test it properly with GitHub action as I'm unable to compile the required Typescript bundle. Could you help me test that?

Sure. First time contributors need to be approved for the CI pipelines to work. I approved you now so the CI CI pipelines should run now on GitHub.

> Apart from that, I believe X11 should not be left out as it is required for building packages, and it can cause problems if the CI environment is different from the official builder's environment.
>
>>https://www.openbsd.org/faq/ports/ports.html#PortsIntro
>
>Building ports on systems without X11 is not supported.
Any idea? :)

Hmm, I didn't know about that. I'm new to OpenBSD. I would like to give that some thought.

Anyway, I think this issue needs to be split into two, one for the 6.9 update and one for the X11. We can handle those separately.

**Comment by ZhanYF** [comment]
Done.

> I approved you now so the CI pipelines should run now on GitHub.

Seems like it's still pending? Anyway it works fine on my repo, I'm not too worried about that.

**Comment by jacob-carlborg** [comment]
> Seems like it's still pending?

Hmm, the original PR has completed the CI pipeline. I'm not sure about the new ones. Perhaps a PR needs to be merged before a new contributor is approved. I've manually started the CI pipeline on the new PRs.

**Comment by jacob-carlborg** [comment]
Closing this in favor of #5 and #6.
